site/content/api/Metrics.md

```diff
@@ -0,0 +1,77 @@
+to which metric.:
+
+| Tag     | Description |
+| ------- | ----------- |
+| host    | The name of the host the service is running on. |
+| type    | The type of the messages being processes. May be `telememtry` or `event`. |
+| tenant  | The name of the tenant. |
+
+### HTTP Metrics
+
+| Metric | Tags | Description |
+| ------ | ---- | ----------- |
+| counter.hono.http.messages.undeliverable.count | host, tenant, type | Messages which could not be processed by the HTTP protocol adapter. Total count since application startup. |
+| meter.hono.http.messages.processed.count | host, tenant, type | Messages processed by the HTTP protocol adapter. Total count since application startup. |
+| meter.hono.http.messages.processed.m1_rate | host, tenant, type | Messages processed by the HTTP protocol adapter. One minute, exponentially weighted, moving average. |
+| meter.hono.http.messages.processed.m5_rate | host, tenant, type | Messages processed by the HTTP protocol adapter. Five minute, exponentially weighted, moving average. |
+| meter.hono.http.messages.processed.m15_rate | host, tenant, type | Messages processed by the HTTP protocol adapter. Fifteen minute, exponentially weighted, moving average. |
+| meter.hono.http.messages.processed.mean_rate | host, tenant, type | Messages processed by the HTTP protocol adapter. Mean rate of messages since the application start. |
+
+### MQTT Metrics
+
+| Metric | Tags | Description |
+| ------ | ---- | ----------- |
+| counter.hono.mqtt.connection.count | host, tenant | Messages processed by the MQTT protocol adapter. Total count since application startup. |
+| counter.hono.mqtt.messages.undeliverable.count | host, tenant, type | Messages which could not be processed by the MQTT protocol adapter- Total count since application startup. |
+| meter.hono.mqtt.messages.processed.count | host, tenant, type | Messages processed by the MQTT protocol adapter. Total count since application startup. |
+| meter.hono.mqtt.messages.processed.m1_rate | host, tenant, type | Messages processed by the MQTT protocol adapter. One minute, exponentially weighted, moving average. |
+| meter.hono.mqtt.messages.processed.m5_rate | host, tenant, type | Messages processed by the MQTT protocol adapter. Five minute, exponentially weighted, moving average. |
+| meter.hono.mqtt.messages.processed.m15_rate | host, tenant, type | Messages processed by the MQTT protocol adapter. Fifteen minute, exponentially weighted, moving average. |
+| meter.hono.mqtt.messages.processed.mean_rate | host, tenant, type | Messages processed by the MQTT protocol adapter. Mean rate of messages since the application start. |
+
+### Messaging Metrics
+
+| Metric | Tags | Description |
+| ------ | ---- | ----------- |
+| counter.hono.messaging.messages.discarded.count | host, tenant, type | Messages which could not be processed by the messaging component. Total count since application startup. |
```

Comment:
We have 'discarded' and 'undeliverable' depending on the settled state (event/telemetry).

Reply:
True. However we discussed that we want to remove the messaging component altogether. So I think we actually should remove the whole section, as we will no longer support this. Right?